### DISCUSSION TITLE
Make a voice notification for bookmarks

### DISCUSSION THREAD

**Discussion by miralas** [organicmaps/organicmaps#8236] [Q&A]
Hi all! I want to make a voice notification to bookmarks, if there are any on the route, like on speed cameras, but I’m new to C++, tell me which direction is better to look?

**Comment by biodranik** [comment]
What does it mean "voice notification to bookmarks"? Can you please elaborate on how it should work from a user's perspective?

**Reply by miralas** [reply]
when the user is driving along a route and there is a bookmark on the route, a voice notification about it sounds 100-200 meters away

**Reply by biodranik** [reply]
If it is about speed cameras, there's a better and easier way: add speed cameras to osm.org, they are already supported and announced by Organic Maps.

**Reply by hb0nd** [reply]
are stops announced?

**Reply by miralas** [reply]
No, I don't need to add new cameras, I need to announce a notification while user drive by route if there is a user mark on route
